src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -183,7 +186,7 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
                 remoteOptions.remoteExecutor,
                 remoteOptions.remoteProxy,
                 authAndTlsOptions,
-                loggingInterceptor);
+                interceptors.toArray(new ClientInterceptor[0]));
```

Comment:
I'd prefer to pass the list in instead of converting it to an array.

Reply:
This was an artifact of using variadic, and was fortunately just conflicted by someone adding another interceptor. Won't appear in the updated patch